com.w1hkj.flcluster.yml

```diff
@@ -9,6 +9,9 @@ finish-args:
   - --device=all
   - --share=network
   - --socket=pulseaudio
+  - --persist=.flcluster
+  - --persist=.fltk
+  - --persist=.nbems
```

Comment:
The `.nbems` directory does not seem to be used by flcluster. You can probably remove it.

Reply:
Yeah, it really looks like that this dir is not used by flcluster. @lu9dce Feel free to remove it.